nomad/acl_endpoint.go

```diff
@@ -79,6 +79,7 @@ func (a *ACL) UpsertPolicies(args *structs.ACLPolicyUpsertRequest, reply *struct
 	if done, err := a.srv.forward("ACL.UpsertPolicies", args, args, reply); done {
 		return err
 	}
+	a.srv.MeasureRPCRate("acl", structs.RateMetricWrite, args)
```

Comment:
Is there any meaningful difference between measuring the RPC rate before or after forwarding? The only thing I can think of is that requests that fail to be forwarded would not be counted, but that's probably OK 🤔

Reply:
Oh, measuring before forwarding would count the request twice (at least). Never mind 😬

Reply:
By doing it after forwarding, we ensure that we only measure a RPC on the server where it's being "finally" processed. 

For example, a write gets measured on the leader, but not the follower (or other region!) it's been forwarded from. Whereas a stale query gets measured on the follower that serves the query. Counting for each hop would be confusing for administrators and would make eventual rate limiting really weird to handle.